atuin/src/command/client/history.rs

```diff
@@ -300,10 +309,20 @@ impl Cmd {
         };
 
         db.update(&h).await?;
+        history_store.push(h).await?;
```

Comment:
it's a bit unfortunate that this can run before InitStore, causing the indexes to be out of order. I guess it doesn't matter for history since it's ordered by the internally stored timestamp anyway